static/app/types/project.tsx

```diff
@@ -120,6 +118,127 @@ export interface TeamWithProjects extends Team {
   projects: Project[];
 }
 
+/**
+ * The type of all platforms keys.
+ * Also includes platforms that cannot be created in the UI anymore.
+ */
+export type PlatformKey =
+  | 'android'
+  | 'apple-ios'
+  | 'apple-macos'
+  | 'apple'
+  | 'capacitor'
+  | 'cordova'
+  | 'dart'
+  | 'dotnet-aspnet'
+  | 'dotnet-aspnetcore'
+  | 'dotnet-awslambda'
+  | 'dotnet-gcpfunctions'
+  | 'dotnet-maui'
+  | 'dotnet-uwp'
+  | 'dotnet-winforms'
+  | 'dotnet-wpf'
+  | 'dotnet-xamarin'
+  | 'dotnet'
+  | 'electron'
+  | 'elixir'
+  | 'flutter'
+  | 'go-echo'
+  | 'go-fasthttp'
+  | 'go-gin'
+  | 'go-http'
+  | 'go-iris'
+  | 'go-martini'
+  | 'go-negroni'
+  | 'go'
+  | 'ionic'
+  | 'java-log4j2'
+  | 'java-logback'
+  | 'java-spring-boot'
+  | 'java-spring'
+  | 'java'
+  | 'javascript-angular'
+  | 'javascript-ember'
+  | 'javascript-gatsby'
+  | 'javascript-nextjs'
+  | 'javascript-react'
+  | 'javascript-remix'
+  | 'javascript-svelte'
+  | 'javascript-sveltekit'
+  | 'javascript-vue'
+  | 'javascript'
+  | 'kotlin'
+  | 'minidump'
+  | 'native-qt'
+  | 'native'
+  | 'node-awslambda'
+  | 'node-azurefunctions'
+  | 'node-connect'
+  | 'node-express'
+  | 'node-gcpfunctions'
+  | 'node-koa'
+  | 'node-serverlesscloud'
+  | 'node'
+  | 'php-laravel'
+  | 'php-symfony2'
+  | 'php'
+  | 'python-aiohttp'
+  | 'python-asgi'
+  | 'python-awslambda'
+  | 'python-bottle'
+  | 'python-celery'
+  | 'python-chalice'
+  | 'python-django'
+  | 'python-falcon'
+  | 'python-fastapi'
+  | 'python-flask'
+  | 'python-gcpfunctions'
+  | 'python-pylons'
+  | 'python-pymongo'
+  | 'python-pyramid'
+  | 'python-quart'
+  | 'python-rq'
+  | 'python-sanic'
+  | 'python-serverless'
+  | 'python-starlette'
+  | 'python-tornado'
+  | 'python-tryton'
+  | 'python-wsgi'
+  | 'python'
+  | 'react-native'
+  | 'ruby-rack'
+  | 'ruby-rails'
+  | 'ruby'
+  | 'rust'
+  | 'unity'
+  | 'unreal'
+  | 'other'
+  // legacy platforms – not included in the create project flow
+  | 'cocoa-objc'
+  | 'cocoa-swift'
+  | 'csharp'
+  | 'dart-flutter'
+  | 'java-android'
+  | 'java-appengine'
+  | 'java-log4j'
+  | 'java-logging'
+  | 'javascript-angularjs'
+  | 'javascript-backbone'
+  | 'javascript-capacitor'
+  | 'javascript-cordova'
+  | 'javascript-electron'
+  | 'native-breakpad'
+  | 'native-crashpad'
+  | 'native-minidump'
+  | 'perl'
+  | 'php-monolog'
+  | 'php-symfony'
+  | 'python-azurefunctions'
+  // TODO(aknaus): check if those are really platform keys and clean up
+  | 'python-tracing'
+  | 'node-tracing'
+  | 'react-native-tracing';
```

Comment:
Those are used to load docs in https://github.com/getsentry/sentry/blob/242fd1dd0654e85bf027df59131db516a0b55395/static/app/components/events/interfaces/spans/inlineDocs.tsx

I doubt that they are real platforms in the DB called like this 🤔